src/scripts/interaction.js

```diff
@@ -1598,6 +1610,16 @@ function Interaction(parameters, player, previousState) {
         });
 
         instance.on('click', function (event) {
+
```

Comment:
Can you have a look at the indentation in the file please.
![image](https://user-images.githubusercontent.com/112154/209638842-58d79213-71a9-4008-9e70-24b8a3d7f8cf.png)

Reply:
Fixed